framework/doc/content/source/transfers/MultiAppInterpolationTransfer.md

```diff
@@ -2,7 +2,16 @@
 
 The MultiAppInterpolationTransfer transfers the source variable to the nearest node on the
 target mesh using mesh interpolation, including the ability to utilize the displaced
-configuration for either or both the source and target.
+configuration for either or both the source and target.  
+
+Nodal transfers using this interpolation scheme is best suited for the interpolation of point cloud data in the source mesh onto a target mesh.  
+The [MultiAppMeshFunctionTransfer](/MultiAppMeshFunctionTransfer.md) may be a better choice for nodal transfers between two meshes because element data will be used in the transfer and the transfer is not dependent on the number of points used in the transfer.  
```

Comment:
I think something is wrong with the documentation of this one too actually. Mentions postprocessors when it s using only variables?

Reply:
(does not have to be fixed here)

Reply:
you are right.  That transfers documentation is really confusing and I think wrong